app/dashboard/page.tsx

```
@@ -173,6 +260,14 @@ export default function DashboardPage() {
           return b.durationSeconds - a.durationSeconds;
         case 'phone':
           return (a.phoneNumber || '').localeCompare(b.phoneNumber || '');
+        case 'sentiment':
+          const sentimentOrder = { positive: 3, neutral: 2, negative: 1 };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)